### DISCUSSION TITLE
ModuleNotFoundError: No module named 'diffusers'

### DISCUSSION THREAD

**Discussion by Riveda72** [lllyasviel/Fooocus#3172] [Q&A] [answered]
Hello, I'm launching Fooocus for the first time using 
`python entry_with_update.py`
and I get this error:
**ModuleNotFoundError: No module named 'diffusers'**

**Comment by ILikeToasters** [comment]
What are you running on and how did you install it? The easiest way on Windows is to just download the 7zip file and extract. Then just run the run.bat file. It will all be set up in the extracted folder.

**Comment by mashb1t** [comment, marked as answer]
Fooocus doesn't use diffusers, so please make sure you're really using this repository, not a fork of it, and if it's a fork please open an issue on their repo. You can find the installation instructions in the readme.
